src/test/kotlin/no/skatteetaten/aurora/gobo/graphql/affiliation/AffiliationQueryTest.kt

```diff
@@ -152,4 +162,22 @@ class AffiliationQueryTest : GraphQLTestWithDbhAndSkap() {
             }
             .graphqlDoesNotContainErrors()
     }
+
+    @Test
+    fun `Query for affiliations with storageGrid objectAreas`() {
+        coEvery { affiliationService.getAllDeployedAffiliations() } returns listOf("paas")
+        coEvery { storageGridObjectAreasService.getObjectAreas(any(), any()) } returns listOf(
+            StoragegridObjectAreaResourceBuilder("aup").build()
```

Comment:
Om jeg tolker denne testen riktig så får man tilbake sgoas til aup for paas, burde det kanskje være lik tilhørighet begge plassene?

Reply:
jeg kan fikse den